user/how-to-guides/how-to-organize-your-qubes.md

```diff
@@ -467,7 +467,7 @@ lessons from our case studies:
   frequent backups now, you'll save yourself from a lot of pain in the future.
   Many people never take backups seriously until they suffer catastrophic data
   loss. That's human nature. If you've experienced that before, then you know
-  the pain. Resolve now never to let it happen again. If you've never
```

Comment:
This isn't a typo. It's already correct. Your change would a bit more controversial, as it splits the infinitive.